app/display/runtime/src/main/java/org/csstudio/display/builder/runtime/app/DisplayRuntimeInstance.java

```diff
@@ -284,13 +285,39 @@ public void loadDisplayFile(final DisplayInfo info)
         // load model off UI thread
         JobManager.schedule("Load Display", monitor ->
         {
-            final DisplayModel model = loadModel(monitor, info);
+            try
+            {
+                final DisplayModel model = loadModel(monitor, info);
 
-            final Future<Void> represented = representation.submit(() -> representModel(model));
-            represented.get();
+                final Future<Void> represented = representation.submit(() -> representModel(model));
+                represented.get();
 
-            // Start runtime for the model
-            RuntimeUtil.startRuntime(model);
+                // Start runtime for the model
+                RuntimeUtil.startRuntime(model);
+            }
+            catch (Exception ex)
+            {
+                logger.log(Level.SEVERE, "Cannot load model from " + info.getPath(), ex);
+
+                final String exception_message;
+
+                if (ex.getCause() != null)
+                    exception_message = ex.getCause().getLocalizedMessage();
+                else
+                    exception_message = "";
```

Comment:
if (ex.getCause() == null) set exception_message = ex.getMessage()?

Reply:
I used `exception_message = ""` because `ex.getMessage()` will be shown by default by the dialog itself and it looked odd that the message was duplicated

Reply:
Yes, the complete message will be shown in the detail section of the dialog as part of the stack trace, if the user opens it.
But your main section messages ends in a colon "...:" to suggest that there's more:

```
Cannot load model from .. path: TheNestedMessage
```

But when there's no nested message, you end with a colon and then there's nothing:
```
Cannot load model from .. path:   <nothing>
```

Reply:
I will create a new pull request removing the colon when there is no cause.